blockchain/src/blockchain/wrappers.rs

```diff
@@ -145,7 +145,7 @@ impl Blockchain {
         if let Ok(account) = self.state.accounts.get(address, None) {
             Some(account)
         } else {
-            warn!("Could not get account for address");
+            warn!("Could not get account for address {}", address.to_user_friendly_address());
```

Comment:
```suggestion
            warn!(address = address.to_user_friendly_address(), "Could not get account for address",);
```
Use tracing style

Reply:
Will it then still show as a user friendly address? Also, what is tracing style?

Reply:
Yes, it will since we're using `to_user_friendly_address`. [This is](https://docs.rs/tracing/latest/tracing/#recording-fields) the tracing style to record fields I was referring to.